src/style.css

```diff
@@ -1 +1,83 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir-Book;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-family: Avenir-Heavy;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+body {
+  margin: 0 8px;
```

Comment:
The margin property should be consistent with the checklist recommendation to avoid potential margin collapse. Consider using only top or bottom margin, but not both.